automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -309,11 +309,11 @@ trait Orchestration extends FireCloudClient with LazyLogging with SprayJsonSuppo
     def createTrialProjects(count: Int): Unit = {
       val campaignManager = UserPool.chooseCampaignManager
       implicit val token: AuthToken = campaignManager.makeAuthToken()
-      val report: TrialProjectReport = countTrialProjects()
-      if (report.available == 0) {
-        postRequest(apiUrl(s"api/trial/manager/projects?operation=create&count=$count"))
+      val trialProjects: TrialProjects = countTrialProjects()
+      if (trialProjects.available < count) {
+        postRequest(apiUrl(s"api/trial/manager/projects?operation=create&count=${count - trialProjects.available}"))
         retry(30.seconds, 10.minutes)({
-          val report: TrialProjectReport = countTrialProjects()
+          val report: TrialProjects = countTrialProjects()
```

Comment:
I'd rename `report` as `trialProjects`, too, for readability and consistency, if possible.